### PR TITLE
Added recipe for pdfminer.six

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/pdfminer.six/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  commands:
+    - pdf2txt.py -h
+    # Can't test latin2ascii.py unless you feed it input.
+    # Can't test dumppdf.py unless you feed it input.
+
+about:
+  home: http://github.com/goulu/pdfminer
+  license: MIT
+  license_family: MIT
+  # license_file: No LICENSE included.
```
Comment (jakirkham) [inline recipes/pdfminer.six/meta.yaml]: Please raise an issue upstream.

Comment (pmlandwehr) [inline recipes/pdfminer.six/meta.yaml]: Done - see [issue 31](https://github.com/goulu/pdfminer/issues/31)

File: recipes/pdfminer.six/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  commands:
+    - pdf2txt.py -h
+    # Can't test latin2ascii.py unless you feed it input.
+    # Can't test dumppdf.py unless you feed it input.
+
+about:
+  home: http://github.com/goulu/pdfminer
+  license: MIT
+  license_family: MIT
+  # license_file: No LICENSE included - see https://github.com/goulu/pdfminer/issues/31
```
Comment (jakirkham) [inline recipes/pdfminer.six/meta.yaml]: xref: https://github.com/goulu/pdfminer/issues/31

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**